src/PdfCoverPage/generator.py

```diff
@@ -0,0 +1,178 @@
+    relevant_fields = extract_required_fields(manifest)
+
+    # generate PDF and get bytes
+    pdf_bytes = build_pdf(relevant_fields)
+    pdf_bytes.seek(0)
+    print(f"generated PDF cover-page for '{identifier}'")
+
+    return send_file(pdf_bytes, mimetype="application/pdf")
+
+
+def extract_required_fields(manifest):
+    """Process manifest and extract only those fields we are interested in"""
+    relevant_fields = {}
+    for k, v in manifest.items():
+        if k == "label":
+            relevant_fields[k] = get_first_lang_value(v)
+        elif k == "homepage":
+            relevant_fields[k] = v[0]["id"]
+        elif k == "metadata":
+            flattened = {}
+            for d in v:
+                label = get_first_lang_value(d["label"])[0]
+                value = get_first_lang_value(d["value"])
+                flattened[label] = value
+            relevant_fields[k] = flattened
+        elif k == "requiredStatement":
+            value = v["value"]
+            relevant_fields[k] = v["value"][get_first_lang(value)]
+        elif k == "provider":
+            relevant_fields[k] = v[0]
+    return relevant_fields
+
+
+def build_pdf(data: dict):
+    """build pdf and return bytes"""
+
+    # configure document
+    pdfmetrics.registerFont(TTFont("Inter", "fonts/Inter-Regular.ttf"))
+    pdfmetrics.registerFont(TTFont("Inter-Bold", "fonts/Inter-Bold.ttf"))
+    styles = getSampleStyleSheet()
+    styles.add(ParagraphStyle(name="Heading", fontName="Inter-Bold", fontSize=12, leading=15))
+    styles.add(ParagraphStyle(name="Footer", fontName="Inter", fontSize=11, leading=13))
+    normal = ParagraphStyle("Normal")
+    normal.fontSize = 12
+    normal.leading = 14
+    normal.fontName = "Inter"
+
+    # build elements
+    pdf_elements = []
+    label_vals = data.get("label", ["---NO TITLE---"])
+    pdf_elements.append(Paragraph(" - ".join(label_vals), styles["Heading"]))
+    pdf_elements.append(Spacer(1, 30))
+
+    def add_metadata(header, value):
+        pdf_elements.append(Paragraph(header, styles["Heading"]))
+        pdf_elements.append(Spacer(1, 6))
+        if isinstance(value, list):
+            pdf_elements.extend([Paragraph(val, normal) for val in value])
+        else:
+            pdf_elements.append(Paragraph(value, normal))
+
+        pdf_elements.append(Spacer(1, 10))
+
+    if metadata := data.get("metadata", {}):
+        if contributors := metadata.get("Contributors", []):
+            add_metadata("Contributors", contributors)
+
+        if pub_creation := metadata.get("Publication/creation", []):
+            add_metadata("Publication/Creation", pub_creation)
+
+    add_metadata("Persistent URL", data["homepage"])
+    pdf_elements.append(Spacer(1, 20))
+
+    # take the first element that is not "Wellcome Collection"
+    if required_statement := [s for s in data.get("requiredStatement", []) if s != "Wellcome Collection"]:
+        add_metadata("License and attribution", required_statement[0])

```

Comment:
Is it correct to take just the first non "Wellcome Collection" value? 

https://github.com/wellcomecollection/platform/issues/4472 has an example where I think we'd want to keep both values:

```json
"rights": "https://creativecommons.org/publicdomain/mark/1.0/",
  "requiredStatement": {
    "label": { "en": [ "Attribution and usage" ] },
    "value": {
      "en": [
        "This material has been provided by The University of Glasgow Library where the originals may be consulted.",
        "<span>This work has been identified as being free of known restrictions under copyright law, including all related and neighbouring rights and is being made available under the <a target=\"_top\" href=\"http://creativecommons.org/publicdomain/mark/1.0/\">Creative Commons, Public Domain Mark</a>.<br/><br/>You can copy, modify, distribute and perform the work, even for commercial purposes, without asking permission.</span>"
      ]
    }
  },
```

Reply:
Ah yes - take all the non "Wellcome Collection" values. This example has two strings because it's from Glasgow University so has an attribution to them as well as the usage statement.

Reply:
For ref: https://github.com/wellcomecollection/iiif-builder/blob/master/src/Wellcome.Dds/Wellcome.Dds.Repositories/Presentation/IIIFBuilderParts.cs#L145